src/plugins/visualizations/public/plugin.ts

```diff
@@ -200,6 +213,7 @@ export class VisualizationsPlugin
       chrome: core.chrome,
       overlays: core.overlays,
       visualizationTypes: types,
+      savedAugmentVisLoader,
```

Comment:
can you update the props passed into the `saved_visualizations.ts` file to include this new one?

Reply:
Thanks for the catch here. I've removed as this was a leftover prop when this loader was being utilized but has been refactored and isn't needed here anymore.